src/terrain/util/transformers.clj

```diff
@@ -40,8 +40,10 @@
   "Generates a set of query parameters to pass to a remote service that requires
    the username of the authenticated user."
   ([]
-     (user-params {}))
+   (user-params {}))
   ([existing-params]
-     (assoc existing-params :user (:shortUsername current-user)))
+   (as-> (add-current-user-to-map {}) m
+         (select-keys m [:user])
+         (merge existing-params m)))
```

Comment:
The `add-current-user-to-map` function will substitute `anonymous` for the `:user` field if `:shortUsername` is empty in `current-user` (updated in #177).